pkgs/top-level/all-packages.nix

```diff
@@ -42238,4 +42238,6 @@ with pkgs;
   ssl-proxy = callPackage ../tools/networking/ssl-proxy { };
 
   code-maat = callPackage ../development/tools/code-maat {};
+
+  orchard = callPackage ../applications/virtualization/orchard { };
```

Comment:
How about switching to the new `by-name` mechanism (https://github.com/NixOS/nixpkgs/pull/237439)?

Reply:
Oh, TIL. I'll do that.